### ISSUE TITLE
Samsung Keyboard copying last word in Chat Input EditText

### ISSUE THREAD

**Issue by cdotindia** [redsolution/xabber-android#774]
Device: Samsung Galaxy A5 (2016) Android Version 6.0.1

You have sent a text message by typing in the Chat Input EditText and pressing send button. Then if you type a new message, the samsung keyboard copies the last word from the previous message and append it in the beginning of your new message. I have only observed this on Samsung Keyboards.

It does not happen when you end the previous message with a punctuation mark (.?!)
![screenshot_20171025-115248](https://user-images.githubusercontent.com/33082161/31983519-307760de-b97b-11e7-879b-e5ff18ea9b6b.png)

**Comment by casperklein** [comment]
Device: S7 (Stock Android 7, Stock Keyboard)

I can confirm this. I noticed that some time ago, but thought that was an exclusive "feature" on my device only 😄.

**Comment by andrewnenakhov** [comment]
Maybe you should post this issue to developers of Samsung keyboard? We don't have plenty of their devices, but on those few we tested Xabber ob, everything worked correctly just like on any Android.

**Comment by andrewnenakhov** [comment]
Just how happened to see a girl with Samsung S8, pressed her into installing Xabber, [didn't see anything wrong](https://photos.app.goo.gl/b2vpJHDdU0GZGkyH2) with Xabber, just atrocious Samsung's 'improvements' to stock android.

**Comment by casperklein** [comment]
I think I found what's causing the problem.

@andrewnenakhov I noticed in your video, that you were not using "auto correct" feature. When I start typing e.g. "he" I see a line below with word suggestions "hello", "hell" etc.
So I disabled that on my device and the issue was gone.

**Comment by andrewnenakhov** [comment]
@casperklein I'd love to test it more, but chasing girls around and pressing them to give me their phones for some testing might get me in trouble. :-D

**Comment by casperklein** [comment]
I fully understand that :D

**Comment by cdotindia** [comment]
@andrewnenakhov @casperklein Yeah disabling "auto-correct" does stop this behaviour but it's not a solution.A user would very much like to have this enabled while using a chat app. Suggested solutions on stackoverflow tell you to try different combinations of inputType params in EditText like `android:inputType="textShortMessage|textAutoCorrect|textCapSentences|textMultiLine"`
but still no luck.

**Comment by andrewnenakhov** [comment]
@ghsrcgh it's hard to fix bug that occurs on some (shitty) manufacturer device without actually having that device. And since our [patreon campaign](https://www.patreon.com/xabber) is going rather... underwhelming, we'd rather concentrate on things that are more important to us. Hope you understand.

But we'll gladly accept pull request with fix to this bug, if you, or someone, will make one.
